runtime/drivers/bigquery/bigquery.go

```diff
@@ -28,53 +28,22 @@ var spec = drivers.Spec{
 	DocsURL:     "https://docs.rilldata.com/connect/data-source/bigquery",
 	ConfigProperties: []*drivers.PropertySpec{
 		{
-			Key:  "google_application_credentials",
-			Type: drivers.FilePropertyType,
-			Hint: "Enter path of file to load from.",
+			Key:         "google_application_credentials",
+			Type:        drivers.InformationalPropertyType,
+			DisplayName: "GCP credentials",
+			Description: "GCP credentials inferred from your local environment.",
+			Hint:        "Set your local credentials: <code>gcloud auth application-default login</code> Click to learn more.",
+			DocsURL:     "https://docs.rilldata.com/connect/data-source/gcs#rill-developer-local-credentials",
 		},
```

Comment:
We should require the user submits the Google Application Credentials as a file, rather than show this message about inferring local credentials. This'll contribute towards [this ticket](https://linear.app/rilldata/issue/PM-4/introduce-explicit-credentials-to-connectors).